src/navigation/TabNavigator.tsx

```diff
@@ -49,37 +52,55 @@ const NavigationRoot = () => {
       <Tab.Screen
         name="Assistant"
         component={Assistant}
-        options={{
-          tabBarLabel: 'Reisesøk',
-          tabBarIcon: ({color}) => <AssistantIcon fill={color} />,
-        }}
+        options={tabSettings('Reisesøk', AssistantIcon)}
       />
       <Tab.Screen
         name="Nearest"
         component={NearbyScreen}
-        options={{
-          tabBarLabel: 'Avganger',
-          tabBarIcon: ({color}) => <Nearby fill={color} />,
-        }}
+        options={tabSettings('Avganger', Nearby)}
       />
       <Tab.Screen
         name="Ticketing"
         component={TicketingScreen}
-        options={{
-          tabBarLabel: 'Billetter',
-          tabBarIcon: ({color}) => <Tickets fill={color} />,
-        }}
+        options={tabSettings('Billetter', Tickets)}
       />
       <Tab.Screen
         name="Profile"
         component={ProfileScreen}
-        options={{
-          tabBarLabel: 'Mitt AtB',
-          tabBarIcon: ({color}) => <Profile fill={color} />,
-        }}
+        options={tabSettings('Mitt AtB', Profile)}
       />
     </Tab.Navigator>
   );
 };
 
 export default NavigationRoot;
+
+type TabSettings = {
+  tabBarLabel(props: {
+    focused: boolean;
+    color: string;
+    position: LabelPosition;
+  }): JSX.Element;
+  tabBarIcon(props: {
+    focused: boolean;
+    color: string;
+    size: number;
+  }): JSX.Element;
+};
+
+function tabSettings(
+  tabBarLabel: string,
+  Icon: (svg: SvgProps) => JSX.Element,
+): TabSettings {
+  return {
+    tabBarLabel: ({color}) => (
+      <Text
+        style={{color, fontSize: 14, lineHeight: 14}}
+        maxFontSizeMultiplier={2}
+      >
```

Comment:
Should we perhaps consider using ThemeText component and type here instead?

Reply:
You're right. I did this initially but found that the default lineHeight didn't work in this case, and ended up doing it manually. But at this point it's easier to use ThemeText and override line height manually. Updated